r/R/compute.R

```diff
@@ -34,6 +48,30 @@ call_function <- function(function_name, ..., args = list(...), options = empty_
   compute__CallFunction(function_name, args, options)
 }
 
+#' List available Arrow C++ compute functions
+#'
+#' This function lists the names of all available Arrow compute functions.
```

Comment:
```suggestion
#' This function lists the names of all available Arrow C++ library compute functions.
```
(also see my comment below)